ec2/spark_ec2.py

```diff
@@ -591,11 +591,15 @@ def launch_cluster(conn, opts, cluster_name):
 
     # AWS ignores the AMI-specified block device mapping for M3 (see SPARK-3342).
     if opts.instance_type.startswith('m3.'):
+        if sys.version_info[0] >= 3:
+            letters = string.ascii_letters
```

Comment:
interesting I didn't realize that - sure I can change this to use `string.ascii_letters` without a conditional check. This does mean we'd potentially break python2.5 or older though